### ISSUE TITLE
hello, the image resolution is poor. how I can improve it? Thanks for your share.

### ISSUE THREAD

**Issue by sxyxf66** [chen0040/keras-text-to-image#4]

**Comment by sxyxf66** [comment]
@chen0040 Hello, Thanks for your share. I learned a lot. The generated image  is still poor when epoch is 30000.....I don't know how to improve it. Would like your reply. Thanks a lot.

**Comment by saisriteja** [comment]
Did you get a good image..... @sxyxf66

**Comment by saisriteja** [comment]
Did you get a good image.....

**Comment by sxyxf66** [comment]
no.The images generated are poor.发自我的华为手机-------- 原始邮件 --------主题：Re: [chen0040/keras-text-to-image] hello, the image resolution is poor. how I can improve it? Thanks for your share. (#4)发件人：saisriteja 收件人：chen0040/keras-text-to-image 抄送：sxyxf66 <13269273379@163.com>,Mention Did you get a good image..... @sxyxf66

**Comment by saisriteja** [comment]
@sxyxf66 did u found any other way to implement text-to-image generation using GAN

**Comment by sxyxf66** [comment]
I found other codes about text-to-image. Their generated images were better. But i don't know how to use my dataset. Because the dataset used in the code are offerred, like "the flower and bird". The Files are complex. 




At 2019-03-27 22:31:48, "saisriteja" <notifications@github.com> wrote:


@sxyxf66 did u found any other way to implement text-to-image generation using GAN

**Comment by Bk073** [comment]
@sxyxf66 How to setup and execute the code ?

**Comment by enes3774** [comment]
Hi, this model isn't enough because it only analyzed about 500 images. But If you want to increase models quality, you can try the following suggestions:

1)You can increase data with data augmentation. For example, you can apply different filters and add this filters name into the text. with this, you can get lots of data.

2)As I said, this model only analyzed 500 images but there are premodel that analyzed millions of images such as "dall-e". I have tested these texts in "dall-e" and the results were amazing.

![indir (9)](https://user-images.githubusercontent.com/77508537/163119338-ea65185a-7a7e-49a2-9a27-68828d3b8067.png)
![indir (8)](https://user-images.githubusercontent.com/77508537/163119368-dd8f19f0-91d1-407b-992c-bf4e91500fb2.png)
![indir (7)](https://user-images.githubusercontent.com/77508537/163119380-5d1831c7-7c51-4d25-8740-ce755ee72881.png)

İf you have questions about this topic. You can contact me at this address: _enesmahmut3774@gmail.com_
